frontend/src/editor/transcription_editor.tsx

```diff
@@ -161,10 +162,10 @@ export function TranscriptionEditor({ documentId }: { documentId: string }) {
       <div className="flex justify-end w-full">
         <SecondaryButton
           className="my-4"
-          onClick={() => {
-            const vtt = generateWebVtt(Automerge.toJS(editor.doc));
-            downloadTextAsFile('document.vtt', 'text/vtt', vtt.toString());
-          }}
+          onClick={() =>
+            showModal(<WebvttExportModal editor={editor} onClose={() => showModal(null)} />)
+          }
+          disabled={editor.doc.children === undefined || !canGenerateVtt(editor.doc.children)}
```

Comment:
I think this is kinda unintuitive. I would whish for at least a hover tooltip why this is disabled

Reply:
or make it selectable and then display a text that it is currently impossible due to missing timings